src/test/java/io/openapitoools/jackson/dataformat/hal/deser/HALBeanDeserializerIT.java

```diff
@@ -34,26 +36,26 @@
     @Test
     public void testDeserialization() throws Exception {
         TopResource tr = om.readValue(new StringReader(HAL_DOC), TopResource.class);
-        assertEquals("1", tr.id);
-        assertEquals("/top/1", tr.self.getHref());
-        assertEquals("/uri/{id}", tr.templated.getHref());
-        assertTrue(tr.templated.getTemplated());
-        assertEquals(2, tr.childLinks.size());
-        assertEquals(2, tr.children.size());
+        Assertions.assertEquals("1", tr.id);
```

Comment:
Couldn't these be kept as static imports? I think this makes for a more readable test.

Reply:
I've changed this to static imports now.